packages/flutter_tools/test/src/runner/flutter_command_test.dart

```diff
@@ -38,12 +44,93 @@ void main() {
     overrides: <Type, Generator>{
       Cache: () => cache,
     });
+
+    testUsingContext('report execution timing by default', () async {
+      final List<int> mockTimes = <int>[1000, 2000];
+      // Crash if called a third time which is unexpected.
+      when(clock.now()).thenAnswer(
+        (Invocation _) => new DateTime.fromMillisecondsSinceEpoch(mockTimes.removeAt(0))
+      );
+
+      final DummyFlutterCommand flutterCommand = new DummyFlutterCommand();
+      await flutterCommand.run();
+      verify(clock.now()).called(2);
+
+      expect(
+        verify(usage.sendTiming(captureAny, captureAny, captureAny, label: captureAny)).captured, 
+        <dynamic>['flutter', 'dummy', const Duration(milliseconds: 1000), null]
+      );
+    },
+    overrides: <Type, Generator>{
+      Clock: () => clock,
```

Comment:
so I ended up defaulting back to a real clock. I think it's more convenient as we move more and more stuff to Clock for tests that don't really care to keep working without null handling and not get an arbitrary value.